entities/build.gradle

```diff
@@ -1,12 +1,10 @@
 plugins {
-    id "com.flipkart.varadhi.java-library-conventions"
+    id "com.flipkart.varadhi.java-published-library-conventions"
 }
 dependencies {
-    implementation(project(":common"))
-    implementation("io.micrometer:micrometer-core")
-    implementation("com.fasterxml.jackson.core:jackson-databind")
-    implementation("jakarta.validation:jakarta.validation-api")
-    implementation('org.hibernate:hibernate-validator:8.0.1.Final')
-    implementation('org.glassfish.expressly:expressly:5.0.0')
-    testImplementation(project(":pulsar"))
+    api("com.fasterxml.jackson.core:jackson-annotations:$jackson_version")
+    api("jakarta.validation:jakarta.validation-api:$jakarta_version")
```

Comment:
why api ?

Reply:
synced offline.
simple rule to follow: if the class is referable due to the exposed methods, then it is part of api.